docs/source/contribute/gsoc_22.rst

```diff
@@ -0,0 +1,69 @@
+Google Summer of Code 2022
+============================================
+
+| **By**, `Omkar Phansopkar <https://github.com/OmkarPh>`_
+| **Project**: `Scancode-Workbench <https://github.com/nexB/scancode-workbench>`_
+| **Organization -** `AboutCode <https://www.aboutcode.org>`_
+
+Overview
+---------------
+ScanCode Workbench provides an advanced visual UI to help you quickly evaluate
+license and other notices identified by 
+`ScanCode <https://github.com/nexB/scancode-toolkit/>`_.
+ScanCode detects licenses, copyrights, packages
+and other interesting information in your code that can be visualised efficiently using workbench.
+
+The GSoC project's goal was to refactor ScanCode Workbench to a React + Typescript implementation and improve various sections of the application including Table view, file uploads, data sync across components, etc.
+Plus, the workbench wasn't updated for a long time, to support the latest scans from scancode-toolkit, hence the schema and UI had to be changed accordingly.
+
+Goals achieved
+-------------------
+
+* Refactor Workbench from vanilla javascript to React + Typescript implementation
+* Update Workbench to support the latest scancode-toolkit output format. (currently supports outputs from scancode-toolkit v31.1.1)
+* Improve User experience in Tableview, Home page, etc
+* Add convenience features like Drop files, History records, etc.
+* Add new sections for Scan Info, package info, etc
+
+
+Quick look
+-------------------
+.. image:: https://user-images.githubusercontent.com/48476025/185114433-7340ee70-57cd-4586-b7e7-964fe736451a.gif
+
+
+Links
+---------------
+* `Download ScanCode Workbench <https://github.com/nexB/scancode-workbench/releases/latest>`_
```

Comment:
We can also add https://github.com/OmkarPh/scancode-workbench/releases/tag/v4.0.0betaPowershell2  and let's make sure we merge your code with all the latest changes when you think it's ready, so we have a release here, in the official repo. But we need not hurry, this is okay.

Reply:
so you mean, lets change that link to release in my fork, and when official release is done, we''ll update this right

Reply:
No let's keep both, no need to come back and update.